File: konzolos/CarsConsole/Car.cs
namespace CarsConsole.Models
{
    public class Car
    {
        public int CarId        { get; set; }
        public string Brand     { get; set; } = string.Empty;
        public string Model     { get; set; } = string.Empty;
        public decimal DailyPrice { get; set; }
    }
}

File: app/Models/Car.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;

class Car extends Model
{
    protected $fillable = [
        'brand',
        'model',
        'license_plate',
        'year',
        'daily_price',
        'user_uid',
    ];

    // egy autóhoz sok foglalás tartozhat
    public function bookings()
    {
        return $this->hasMany(Booking::class);
    }
}
